src/create-config.ts

```diff
@@ -0,0 +1,317 @@
+
+  //// Build Config.json  ////
+
+  // Get all the chains the API will be deployed on
+  const apiChains = [
+    ...new Set(
+      Object.keys(apiData.beacons).flatMap((beacon) => apiData.beacons[beacon].chains.map((chain) => chain.name))
+    ),
+  ];
+
+  const chains = apiChains.map((chainName) => {
+    const chainId = chainNameToChainId[chainName];
+    //TODO: Add RequesterAuthorizerWithManager
+    const authorizers =
+      response.airnodeAuthorizer && RequesterAuthorizerWithAirnodeAddresses[chainId]
+        ? [RequesterAuthorizerWithAirnodeAddresses[chainId]]
+        : [];
+    const contracts = { AirnodeRrp: AirnodeRrpAddresses[chainId] || '' };
+    const options = {
+      txType: 'eip1559' as const,
+      priorityFee: {
+        value: 3.12,
+        unit: 'gwei' as const,
+      },
+      baseFeeMultiplier: 2,
+    };
+    const providers = {
+      provider1: {
+        url: `\${${chainName}_PROVIDER_URL}`.toUpperCase(),
+      },
+    };
+
+    return {
+      authorizers,
+      contracts,
+      id: `${chainId}`,
+      providers,
+      type: 'evm' as const,
+      options,
+      maxConcurrency: 100,
+      blockHistoryLimit: 300,
+      minConfirmations: 0,
+    };
+  });
+
+  const nodeSettings = {
+    nodeVersion: require('@api3/airnode-node/package.json').version,
+    cloudProvider: {
+      type: 'aws' as const,
+      region: 'us-east-1' as const,
+      disableConcurrencyReservations: true,
+    },
+    airnodeWalletMnemonic: '${AIRNODE_WALLET_MNEMONIC}',
+    heartbeat: {
+      enabled: response.airnodeHeartbeat,
+      ...(response.airnodeHeartbeat && {
+        apiKey: '${HEARTBEAT_API_KEY}',
+        id: '${HEARTBEAT_ID}',
+        url: '${HEARTBEAT_URL}',
+      }),
+    },
+    httpGateway: {
+      enabled: true,
+      apiKey: '${HTTP_GATEWAY_API_KEY}',
+      maxConcurrency: 20,
+    },
+    httpSignedDataGateway: {
+      enabled: true,
+      apiKey: '${HTTP_SIGNED_DATA_GATEWAY_API_KEY}',
+      maxConcurrency: 20,
+    },
+    logFormat: 'plain' as const,
+    logLevel: 'INFO' as const,
+    stage: 'dev' as const,
+    skipValidation: true,
+  };
+
+  //generate the triggers from the OISes
+  const oisTriggers = await Promise.all(
+    Object.values(apiData.ois).flatMap((ois) => {
+      return ois.endpoints.map(async (endpoint) => ({
+        endpointId: await deriveEndpointId(ois.title, endpoint.name),
+        endpointName: endpoint.name,
+        oisTitle: ois.title,
+      }));
+    })
+  );
+
+  const triggers = {
+    rrp: oisTriggers,
+    http: oisTriggers,
+    httpSignedData: oisTriggers,
+  };
+
+  const apiCredentials = Object.values(apiData.ois).flatMap((ois) => {
+    return Object.keys(ois.apiSpecifications.components.securitySchemes).map((security) => {
+      return {
+        oisTitle: ois.title,
+        securitySchemeName: security,
+        securitySchemeValue: `\${SS_${security.toUpperCase()}}`.replace(/ /g, '_'),
+      };
+    });
+  });
+
+  const Config: Config = {
```

Comment:
So, because you're including this object in `OperationsRepository` and `OperationsRepository` is fully typed, you can actually remove this type too, as well as `AirkeeperConfig` further up.